pandas/tests/formats/test_format.py

```diff
@@ -3925,6 +3925,10 @@ def test_rounding(self):
         result = formatter(0)
         self.assertEqual(result, u(' 0.000'))
 
+    def test_nan(self):
+        formatter = fmt.EngFormatter(accuracy=1, use_eng_prefix=True)
+        result = formatter(np.nan)
+        self.assertEqual(result, u('NaN'))
 
```

Comment:
Could you please tell me what you meant by "its just a smoke test, you don't need to compare it"